main/store/migrations/index.js

```diff
@@ -536,7 +536,14 @@ const migrations = {
         secondary: removeOptimismConnection(optimism.connection.secondary)
       }
     }
-
+    return initial
+  },
+  26: (initial) => {
+      Object.values(initial.main.networks.ethereum).forEach((network) => {
+      if(Object.hasOwn(network, "symbol")){
```

Comment:
do we need this check?

Reply:
Doesn't make any functional difference, will remove it